reminders/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Reminders</title>
+        <link rel="stylesheet" type="text/css" href="style.css"/>
+    </head>
+    <body>
+        <div class="reminders-container">
+            <h1>To-Do's</h1>
+            <p>Add item
+                <input id="list-items">
+                <button>Add item</button>
+            </p>
+            <hr>
+            <div>
+                <input type="checkbox"> Go grocery shopping <br>
+                <input type="checkbox"> Work out <br>
+                <input type="checkbox"> Do laundry <br>
+                <input type="checkbox"> Finish this project<br>
+                <input type="checkbox"> Play guitar
```

Comment:
I see you're having to use `<br>` tags after each to-do item, is there something else you can use to maybe make the element take up the whole width by itself?